Analysis/config/high_mass_analysis.py

```diff
@@ -980,9 +983,11 @@ def makeOneLegInputs(label,obj,inputs):
 
 # load appropriate scale and smearing bins here
 # systematics customization scripts will take care of adjusting flashggDiPhotonSystematics
-process.load('flashgg.Systematics.escales.escale76X_16DecRereco_2015')
+#process.load('flashgg.Systematics.escales.escale76X_16DecRereco_2015')
```

Comment:
can you please put a switch based on the dataset name?
something like 
if "Run2015" in customize.options.datasetName() or "76X" in customize.options.datasetName():
     process.load('flashgg.Systematics.escales.escale76X_16DecRereco_2015')
else:
    process.load('flashgg.Systematics.escales.test_2016B_corr')

Thanks.
P.